src/docsUrl.js

```diff
@@ -0,0 +1,10 @@
+const repoUrl = 'https://github.com/benmosher/eslint-plugin-import'
+
+export default function docsUrl(ruleName, commitHash) {
+  let baseUrl = `${repoUrl}/tree/master/docs/rules`
```

Comment:
This can be `blob/master` — `tree` is used for directories, while `blob` is used for files.

Reply:
Wow, it looks like this is a pretty common issue since GitHub automatically redirects the wrong link, but you are absolutely correct.